docs/tutorials/create_architecture.md

```diff
@@ -1,221 +1,225 @@
-path.summary.pprint()
+   ```python
+   path = paths[0]
+   path.summary.pprint()
 
----以下出力---
+   ---Output text as below---
 
-path:
-  - message_context: null # ノードレイテンシの定義。
-    node: /sensor_dummy_node
-  - topic: /topic1
-  - message_context:
-      publisher_topic_name: /topic2
-      subscription_topic_name: /topic1
-      type: callback_chain
-    node: /filter_node
-  - topic: /topic2
-  - message_context: null
-    node: /message_driven_node
-  - topic: /topic3
-  - message_context: null
-    node: /timer_driven_node
-  - topic: /topic4
-  - message_context: null
-    node: /actuator_dummy_node
-```
+   path:
+     - message_context: null # for definition of node latency
+       node: /sensor_dummy_node
+     - topic: /topic1
+     - message_context:
+         publisher_topic_name: /topic2
+         subscription_topic_name: /topic1
+         type: callback_chain
+       node: /filter_node
+     - topic: /topic2
+     - message_context: null
+       node: /message_driven_node
+     - topic: /topic3
+     - message_context: null
+       node: /timer_driven_node
+     - topic: /topic4
+     - message_context: null
+       node: /actuator_dummy_node
+   ```
 
-決定したパスに`target_path`と名前を付け、アーキテクチャファイルに保存します。
+4. Give a name to selected path and update architecture file
 
-```python
-arch.add_path('target_path', path)
-arch.export('./architecture.yaml')
-```
+   ```python
+   arch.add_path('target_path', path)
+   arch.export('./architecture.yaml', force=True)
+   ```
 
-アーキテクチャファイルには、以下のように記載されます。
+   The updated architecture file describes the path named as `target_path`.
 
-```yaml
-named_paths:
-  - path_name: target_path
-    node_chain:
-      - node_name: /sensor_dummy_node
-        publish_topic_name: /topic1
-        subscribe_topic_name: UNDEFINED
-      - node_name: /filter_node
-        publish_topic_name: /topic2
-        subscribe_topic_name: /topic1
-      - node_name: /message_driven_node
-        publish_topic_name: /topic3
-        subscribe_topic_name: /topic2
-      - node_name: /timer_driven_node
-        publish_topic_name: /topic4
-        subscribe_topic_name: /topic3
-      - node_name: /actuator_dummy_node
-        publish_topic_name: UNDEFINED
-        subscribe_topic_name: /topic4
-```
+   ```yaml
+   named_paths:
+     - path_name: target_path
+       node_chain:
+         - node_name: /sensor_dummy_node
+           publish_topic_name: /topic1
+           subscribe_topic_name: UNDEFINED
+         - node_name: /filter_node
+           publish_topic_name: /topic2
+           subscribe_topic_name: /topic1
+         - node_name: /message_driven_node
+           publish_topic_name: /topic3
+           subscribe_topic_name: /topic2
+         - node_name: /timer_driven_node
+           publish_topic_name: /topic4
+           subscribe_topic_name: /topic3
+         - node_name: /actuator_dummy_node
+           publish_topic_name: UNDEFINED
+           subscribe_topic_name: /topic4
+   ```
 
 ## ノードレイテンシの定義指定
```

Comment:
I left a sentence in Japanese, and I'll remove it.

Reply:
fixed https://github.com/tier4/CARET_doc/pull/36/commits/8e5196debdafc6a3762d802af144337303de44ad